ovirt-populator/ovirt-populator.go

```diff
@@ -9,133 +9,54 @@ import (
 	"os"
 	"os/exec"
 
-	"github.com/konveyor/forklift-controller/pkg/apis/forklift/v1beta1"
-	populator_machinery "github.com/kubev2v/lib-volume-populator/populator-machinery"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
-	"k8s.io/apimachinery/pkg/runtime"
 	"k8s.io/apimachinery/pkg/runtime/schema"
 	"k8s.io/client-go/dynamic"
 	"k8s.io/client-go/kubernetes"
 	"k8s.io/client-go/rest"
 	"k8s.io/klog/v2"
 )
 
-const (
-	prefix     = "forklift.konveyor.io"
-	mountPath  = "/mnt/"
-	devicePath = "/dev/block"
-)
+type engineConfig struct {
+	URL      string
+	username string
+	password string
+	ca       string
+}
+
+type TransferProgress struct {
+	Transferred uint64  `json:"transferred"`
+	Description string  `json:"description"`
+	Size        *uint64 `json:"size,omitempty"`
+	Elapsed     float64 `json:"elapsed"`
+}
 
 const (
 	groupName  = "forklift.konveyor.io"
 	apiVersion = "v1beta1"
-	kind       = "OvirtVolumePopulator"
 	resource   = "ovirtvolumepopulators"
 )
 
 func main() {
-	var mode, engineUrl, secretName, diskID, volPath, crName, crNamespace, httpEndpoint, metricsPath, masterURL, kubeconfig, imageName, namespace string
-
-	// Main arg
-	flag.StringVar(&mode, "mode", "", "Mode to run in (controller, populate)")
+	var engineUrl, secretName, diskID, volPath, crName, crNamespace, masterURL, kubeconfig, namespace string
```

Comment:
`masterURL` and `kubeconfig` not used. need to check whether it is needed once the image works.